doc/usage.rst

```diff
@@ -151,6 +151,17 @@ When the post does not exist, you are using ``abort()`` to stop the request
 early. It actually throws an exception, which you will see how to handle later
 on.
 
+You can also use regular expressions to control how variable parts of a route
+definition are matched:
```

Comment:
You need to add a double `:` like ``::` to indicate the indented section is a code block.
See https://github.com/silexphp/Silex/pull/1110/files#diff-665ffb7d19093ce3b903440e23eb6e98R169